### PR TITLE
Update reviewing.md to reflect CNTI

### PR DESCRIPTION
**Pull request by martin-mat**
Refs lfn-cnti#77

## Pre-submission checklist:

Please check each of these after submitting your pull request:

- [ ] If this is a new entry, have you submitted a signed [participation form](https://github.com/cncf/cnf-certification/blob/main/Certified_CNF_Form.md)?
- [ ] Did you include the product/project logo in SVG, EPS or AI format?
- [ ] Does your logo clearly state the name of the product/project/org and follow the other logo [guidelines](https://github.com/cncf/landscape#logos)?
- [ ] If your product/project is open source, did you include the repo_url?
- [ ] Did you copy and paste the [installation and configuration instructions](https://github.com/cncf/k8s-conformance/blob/master/faq.md#can-i-provide-a-link-to-the-installation-directions) into the README.md file in addition to linking to them?

### CODE REVIEW

**Review by taylor — APPROVED** [review]
lgtm

File: reviewing.md [hunk 1]
```diff
@@ -1,26 +1,26 @@
...
-1. Verify that the Certification Version is the current or previous version of CNF Certification.
-1. Look at results log, cnf-testsuite-results-YYYY-MMDD-HHMMSS-NNN.yml. Verify that the major.minor component of the CNF Certification match the certification version being claimed. The patch version does not matter.
+1. Verify that the list of files matches the [expected list](https://github.com/lfn-cnti/certification/blob/main/instructions.md#contents-of-the-pr).
+1. Note the vX.Y subdirectory that the PR is in, this is the version of CNTI Certification for which certified is being claimed, referenced as the "Certification Version" from hereon.
+1. Verify that the Certification Version is the current or previous version of CNTI Certification.
+1. Look at results log, cnf-testsuite-results-YYYY-MMDD-HHMMSS-NNN.yml. Verify that the major.minor component of the CNTI Certification matches the certification version being claimed. The patch version does not matter.
 1. Verify that the RESULTS SUMMARY section of results log "X of Y essential tests passed". There should be at least 10 essential tests passed.
```
Comment (lixuna) [inline reviewing.md:9]: FYI - this sentence "There should be at least 10 essential tests passed" will be updated in the next 1-2 weeks when the v2.0 certification list of tests is updated.

File: reviewing.md [hunk 2]
```diff
@@ -1,26 +1,26 @@
...
 3. Verify the product name, product version and website_url are listed in the "Qualifying Offerings" section of the vendor's Participation Form.
 
-If the submission doesn't meet all policy requirements, reply with a message indicating "Signed participation form needed", "Files missing from PR", "Membership in CNCF or confirmation of non-profit status needed", etc.
+If the submission doesn't meet all policy requirements, reply with a message indicating "Signed participation form needed", "Files missing from PR", "Membership in LFN or confirmation of non-profit status needed", etc.
 
 ### Tasks to Complete After Review
 1. Update the Certified CNF spreadsheet to reflect the vendor's certified offering.
-2. Add the vendor's information to the CNCF landscape which also causes them to appear on https://www.cncf.io/certification/cnf.
-3. Add a comment saying "This product is now a "Certified CNF".
+2. Add the vendor's information to the LFN-CNTI landscape which also causes them to appear on [LFN-CNTI Landing Page](https://lfnetworking.org/cloud-native-telecom-initiative).
```
Comment (lixuna) [inline reviewing.md:23]:
```suggestion
2. Add the vendor's information to the [LFN-CNTI Landing Page](https://lfnetworking.org/cloud-native-telecom-initiative).
```

File: reviewing.md [hunk 3]
```diff
@@ -1,26 +1,26 @@
...
+3. Add a comment saying "This product is now a "Certified LFN-CNTI".
```
Comment (lixuna) [inline reviewing.md:24]:
```suggestion
3. Add a comment saying "This product is now "CNTI Certified."
```

**Review by lixuna — CHANGES_REQUESTED** [review]
changes requested

**Review by lixuna — APPROVED** [review]
lgtm

**Pull request merged**